bin/expressive-module

```diff
@@ -22,6 +24,8 @@ if (file_exists($a = __DIR__ . '/../../../autoload.php')) {
     exit(1);
 }
 
-$command = new Command($argv[0], new ConsoleHelper());
-$return = $command->process(array_slice($argv, 1));
-exit($return);
+$application = new Application('expressive-module', VERSION);
+$application->add(new CreateCommand('create'));
```

Comment:
Yes. Since we already have documented migration steps, I cannot break how these original scripts worked; hence the difference between command names and what commands are exposed.